tests/integration/test_charm.py

```diff
@@ -17,22 +17,25 @@ async def test_build_and_deploy(ops_test: OpsTest):
     openldap_charm = await ops_test.build_charm(".")
     await asyncio.gather(
         ops_test.model.deploy("postgresql-k8s", application_name=PSQL, num_units=1),
-        ops_test.model.deploy(openldap_charm,
-                              resources={'openldap-image': 'openldapcharmers/openldap:2.4.50'},
-                              application_name=APP_NAME, num_units=1),
+        ops_test.model.deploy(
+            openldap_charm,
+            resources={'openldap-image': 'openldapcharmers/openldap:2.4.50'},
+            application_name=APP_NAME,
+            num_units=1,
+        ),
     )
     await ops_test.model.wait_for_idle(apps=[APP_NAME, PSQL])
     assert ops_test.model.applications[APP_NAME].status == "waiting"
     assert ops_test.model.applications[PSQL].status == "active"
 
-    await ops_test.model.add_relation(APP_NAME, PSQL)
+    await ops_test.model.add_relation(APP_NAME + ":db", PSQL + ":db")
     await ops_test.model.wait_for_idle(apps=[APP_NAME, PSQL])
     assert ops_test.model.applications[APP_NAME].status == "active"
     assert ops_test.model.applications[PSQL].status == "active"
 
 
 @pytest.mark.abort_on_fail
-async def test_blocks_without_zookeeper(ops_test: OpsTest):
+async def test_maintenance_without_postgresql(ops_test: OpsTest):
     await asyncio.gather(ops_test.model.applications[PSQL].remove())
     await ops_test.model.wait_for_idle(apps=[APP_NAME])
-    assert ops_test.model.applications[PSQL].status == "blocked"
+    assert ops_test.model.applications[APP_NAME].status == "waiting"
```

Comment:
How about `WaitingStatus.name`instead of using the string `waiting`

Reply:
This is good, thanks! I took the example from https://github.com/canonical/kafka-k8s-operator/blob/main/tests/integration/test_charm.py  Is there any recommendation on what to use nowadays? I like your proposal as it adds more structure (compared just to a string).

Reply:
I'm not aware of any other

Reply:
I've added the change, look better that way, thanks :)